src/discord/staff/events.py

```diff
@@ -109,49 +110,52 @@ async def event_remove(
         )
 
         # Check to make sure event has previously been added
-        event_not_in_list = event_name not in [
-            e["name"] for e in src.discord.globals.EVENT_INFO
-        ]
+        event = next(
+            (e for e in src.discord.globals.EVENT_INFO if e.name == event_name),
+            None,
+        )
 
         # Check to see if role exists on server
         server = self.bot.get_guild(env.server_id)
         potential_role = discord.utils.get(server.roles, name=event_name)
 
-        if event_not_in_list and potential_role:
```

Comment:
I assume that this was *supposed* to be `event_not_in_list and not potential_role` given the comment below it was:
```python
# If no event in list and no role exists on server
```